sofdev6-02-2024/TheBoys
# feat: View All Community and see details

The view to see the communities and that users can subscribe to this, once subscribed can enter the community and begin to see its content.

Steps to test it: 

1. create one or more communities.
2. you must go to http://localhost:3000/communitiesUser , to see the user's communities.
3. you will be able to see the communities and the ones you have already subscribed to.
4. subscribe to a community
5. now if you go back to the page you will be able to see your community that you have just successfully subscribed to. 
6. enter a community you are already a part of and you will be able to see the messages that are in it.
7. now you can write messages and send images or gifs.
